source/_integrations/velux.markdown

```diff
@@ -62,7 +64,7 @@ Remember: You must complete the configuration within 5 minutes of rebooting the
 
 ### Action `velux.reboot_gateway`
 
-Reboots the configured KLF 200 gateway.
+Reboots the configured KLF 200 gateway. _This action will be deprecated in a future release. To reboot your gateway, use the **Reboot** button in Home Assistant's UI for the gateway device._
```

Comment:
We can just remove this whole section when we deprecate the action

Reply:
Yepp: https://github.com/home-assistant/home-assistant.io/pull/41744